purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -677,7 +701,7 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     @JvmSynthetic
     internal fun postAttributionData(
         jsonObject: JSONObject,
-        network: AttributionNetwork,
+        network: com.revenuecat.purchases.common.attribution.AttributionNetwork,
```

Comment:
Have to do this because this class references both `AttributionNetwork` classes. I could just rename the `AttributionNetwork` inside `common`

Reply:
again, I think disambiguation at the import level might be better
```kotlin
import com.revenuecat.purchases.common.attribution.AttributionNetwork as CommonAttributionNetwork
.
.
.
network: CommonAttributionNetwork
```